util/DateOperations.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * Copyright (C) 2022 Igalia, S.L. <info@igalia.com>
+ *
+ * This file is part of PhpReport.
+ *
+ * PhpReport is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PhpReport is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PhpReport.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace Phpreport\Util;
+
+class DateOperations
+{
+    public static function dateBelongsToPeriod(\DateTime $date, \DateTime $init, \DateTime $end): bool
```

Comment:
Some time ago we added `JourneyHistoryVO:: dateBelongsToJourney`. We [mentioned](https://github.com/Igalia/phpreport/pull/515#pullrequestreview-762029898) that it would be useful to move it to the parent class `BaseHistoryVO`. Doing that, we wouldn't need this new class...

Reply:
That's true, I completely forgot about that. Let me move to `BaseHistoryVO` instead

Reply:
Done